README.md

```diff
@@ -502,6 +502,47 @@ export default Base.extend({
 });
 ```
 
+### Upgrading to 2.0
```

Comment:
As 2.0 isn't released yet, I'd maybe call this _"Deprecation of Authorizers"_ or so. We should also add a warning and a link to this section at the top of the _"Authorizers"_ section.